Description: Upstream changes introduced in version 40-0-0
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libarchive-zip-perl (40-0-0) unstable; urgency=low
 .
   * Port for NCP4
   * Used new version scheme
 .
 The person named in the Author field signed this changelog entry.
Author: Peter Zaickin <peter.zaickin@nexenta.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ libarchive-zip-perl-40-0/test.log
@@ -0,0 +1,83 @@
+zipinfo output:
+$ZIP = bless( {
+  "eocdOffset" => 0,
+  "fileName" => "",
+  "writeEOCDOffset" => 0,
+  "diskNumberWithStartOfCentralDirectory" => 0,
+  "writeCentralDirectoryOffset" => 0,
+  "centralDirectorySize" => 65,
+  "members" => [],
+  "numberOfCentralDirectoriesOnThisDisk" => 1,
+  "zipfileComment" => "",
+  "numberOfCentralDirectories" => 1,
+  "diskNumber" => 0,
+  "centralDirectoryOffsetWRTStartingDiskNumber" => 349
+}, 'Archive::Zip::Archive' );
+
+Found EOCD at 414 (0x19e)
+
+Found central directory for member #1 at 349
+$CDMEMBER1 = bless( {
+  "externalFileName" => "(testin.zip)",
+  "uncompressedSize" => 300,
+  "fileName" => "testdir/testing.txt",
+  "versionNeededToExtract" => 20,
+  "fileAttributeFormat" => 3,
+  "diskNumberStart" => 0,
+  "compressionMethod" => 0,
+  "eocdCrc32" => "2889301810",
+  "fileComment" => "",
+  "externalFileAttributes" => "2176188416",
+  "internalFileAttributes" => 0,
+  "bitFlag" => 0,
+  "lastModFileDateTime" => "1054710059",
+  "crc32" => "2889301810",
+  "versionMadeBy" => 20,
+  "localExtraField" => "",
+  "localHeaderRelativeOffset" => 0,
+  "possibleEocdOffset" => undef,
+  "desiredCompressionMethod" => 0,
+  "compressedSize" => 300,
+  "desiredCompressionLevel" => 0,
+  "dataOffset" => 0,
+  "fh" => $Archive::Zip::BufferedFileHandle,
+  "isSymbolicLink" => 0,
+  "cdExtraField" => ""
+}, 'Archive::Zip::ZipFileMember' );
+
+
+$LHMEMBER1 = bless( {
+  "externalFileName" => "(testin.zip)",
+  "uncompressedSize" => 300,
+  "fileName" => "testdir/testing.txt",
+  "versionNeededToExtract" => 20,
+  "fileAttributeFormat" => 3,
+  "diskNumberStart" => 0,
+  "compressionMethod" => 0,
+  "eocdCrc32" => "2889301810",
+  "fileComment" => "",
+  "externalFileAttributes" => "2176188416",
+  "internalFileAttributes" => 0,
+  "bitFlag" => 0,
+  "lastModFileDateTime" => "1054710059",
+  "crc32" => "2889301810",
+  "versionMadeBy" => 20,
+  "localExtraField" => "",
+  "localHeaderRelativeOffset" => 0,
+  "possibleEocdOffset" => undef,
+  "desiredCompressionMethod" => 0,
+  "compressedSize" => 300,
+  "desiredCompressionLevel" => 0,
+  "dataOffset" => 49,
+  "fh" => $Archive::Zip::BufferedFileHandle,
+  "isSymbolicLink" => 0,
+  "cdExtraField" => ""
+}, 'Archive::Zip::ZipFileMember' );
+
+End of member: 349, CD at 349
+
+ziptest output:
+ Length    Size         Last Modified         CRC-32  Name
+-------- --------  ------------------------  -------- ----
+     300      300  Wed Jun 29 19:09:22 2011  ac373f32 testdir/testing.txt
+All CRCs check OK
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/fileC.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/file2.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/string2.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/stringU2.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/fileC2.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/string.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/stringU.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/testdir/file.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
--- /dev/null
+++ libarchive-zip-perl-40-0/extracted/testdir/testing.txt
@@ -0,0 +1,102 @@
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+11
+12
+13
+14
+15
+16
+17
+18
+19
+20
+21
+22
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+45
+46
+47
+48
+49
+50
+51
+52
+53
+54
+55
+56
+57
+58
+59
+60
+61
+62
+63
+64
+65
+66
+67
+68
+69
+70
+71
+72
+73
+74
+75
+76
+77
+78
+79
+80
+81
+82
+83
+84
+85
+86
+87
+88
+89
+90
+91
+92
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
